components/event-publisher-proxy/pkg/sender/jetstream.go

```diff
@@ -18,9 +19,12 @@ import (
 )
 
 const (
-	natsBackend           = "nats"
-	jestreamHandlerName   = "jetstream-handler"
-	noSpaceLeftErrMessage = "no space left on device"
+	natsBackend          = "nats"
+	jetstreamHandlerName = "jetstream-handler"
+	// jetstreamThreshold represents the number of bytes which is approximately equal to 930MiB
+	// 930MiB is the approximate threshold when the JetStream storage of 1GiB gets full and JetStream gets disabled.
+	jetstreamThreshold             = 976000000
```

Comment:
I have some recommendations here:
- The `jetstreamThreshold` number is hard to read, how about instead of using actual bytes as the threshold, we use percentage and we calculate the actual bytes ourselves. Also it should be configured
- The total storage of 1G is a hidden info and should be configured.

For example, we could have the following configurations passed to the JetStream publisher:
- `nats.storage.size: 1Gi`
- `nats.storage.threshold: 50%`

Reply:
Let's discuss the possible options with the team